handler/handler_test.go

```diff
@@ -210,16 +210,20 @@ func TestHandleWithBothCreateAndUpdateFailedESResponse(t *testing.T) {
 	})
 }
 
-func TestHandleWithCreateAndInternalServerESResponse(t *testing.T) {
+func TestHandleWithCreate409ErrorAndInternalServerESResponse(t *testing.T) {
```

Comment:
I would have create a new test, not replace this one

Reply:
thanks for the comment. I have updated the handler to accomodate another scenario of 500 error while creating in the first instance. Test already exists to check both the scenarios (create with 409 and 500).